geoips/commandline/args.py

```diff
@@ -635,6 +635,15 @@ def add_args(parser, arglist=None):
                     be passed to the algorithm in full.""",
         )
 
+    if arglist is None or "image_compare_threshold" in arglist:
+        procflow_group.add_argument(
+            "--image_compare_threshold",
+            choices=["lenient", "medium", "strict"],
+            default="medium",
+            type=str.lower,
+            help="Threshold Value for comparing Pixels in Images.",
+        )
+
```

Comment:
I'm wondering if, rather than creating an entirely new command line argument specifically for image_compare_threshold, if we just want to pass this in directly as output_checker_kwargs ?  We want the "kwargs" option to output_checker plugins anyway, probably ?  Thoughts?

Reply:
Done!